src/Model/Domain/ClickTracking.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * Copyright (C) 2013 Mailgun
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+namespace Mailgun\Model\Domain;
+
+/**
+ * Represents a single Click Tracking setting for a domain tracking.
+ *
+ * @author Artem Bondarenko <artem@uartema.com>
+ */
+final class ClickTracking
+{
+    private $active;
+
+    public static function create(array $data): self
+    {
+        $model = new self();
+        $active = $data['active'] ?? null;
+
+        if (true === $active) {
+            $model->active = 'yes';
+        } elseif (false === $active) {
+            $model->active = 'no';
+        } else {
+            $model->active = $active;
+        }
```

Comment:
Suggesting a ternary operator instead of concatenating a set of conditionals, as it makes the code shorter

```php
        $active = $data['active'] ?? null;

        $model = new self();
        $model->active = 'htmlonly' === $active ? $active : ($active ? 'yes' : 'no');
```